bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestBotFactory.java

```diff
@@ -244,6 +244,10 @@ public List<Bot> create(BotConfiguration configuration) {
                 botBuilder.approval(approval);
             }
 
+            if (repo.value().contains("fixVersionOverride")) {
```

Comment:
I think the config option should reflect that we are turning off this warning specifically, rather than that we have configured a different fixVersion in the notifier. There are more situations where this warning doesn't apply (see bug comments).

How about `versionMismatchWarning` and I think we should even default it to `false` and selectively add it to the repos we want it on.

Reply:
Sure, thanks